Disable reading $HOME/.config/pango/pangorc if building for the target.

patch by Jungshik Shin (jungshik@google) <jshin@chromium.org> adopted
from jorgelo's patch for pango-1.28 
(see  https://chromium-review.googlesource.com/#/c/45316 ).

TEST='emerge-$BOARD pango' applies the patch and pango builds.
TEST='USE=cros_host emerge-$BOARD pango' does not apply the patch
TEST='sudo emerge pango' does not apply the patch

diff --git a/pango/pango-utils.c b/pango/pango-utils.c
index 18ffa26..a53222b 100644
--- a/pango/pango-utils.c
+++ b/pango/pango-utils.c
@@ -624,6 +624,7 @@ read_config (void)
 
       tmp_hash = read_config_system ();
 
+      /*
       filename = g_build_filename (g_get_user_config_dir (),
                                    "pango",
                                    "pangorc",
@@ -634,7 +635,7 @@ read_config (void)
       envvar = g_getenv ("PANGO_RC_FILE");
       if (envvar)
         read_config_file (envvar, TRUE, tmp_hash);
-
+      */
       g_once_init_leave ((gsize*)&config_hash, (gsize)tmp_hash);
     }
 
